q2_diversity/plugin_setup.py

```diff
@@ -723,7 +725,7 @@
     },
     parameter_descriptions={
         'metrics': ('The metrics to be measured. By default computes '
-                    'observed_otus, shannon, and if phylogeny is '
+                    'observed_features, shannon, and if phylogeny is '
```

Comment:
Please revert - I don't believe we have changed the name of this metric here, yet.

```suggestion
                    'observed_otus, shannon, and if phylogeny is '
```

Bigger picture - we will just have references to both `observed_otus` and `observed_features` in this plugin until we sort everything out.

Reply:
Sorry for the noise. That shouldn't have been changed.